src/utils/DockerUtil.js

```diff
@@ -76,8 +76,45 @@ export default {
     } else {
       startopts.PublishAllPorts = true;
     }
+    // Add support for linked containers and runtime settings
+    if (containerData.HostConfig) {
+      var hostConfig = containerData.HostConfig;
+      if(typeof hostConfig.CpuShares !== "undefined" && hostConfig.CpuShares !== 0){
```

Comment:
Replace `typeof hostConfig.CpuShares !== "undefined"` with `hostConfig.CpuShares` and similar below.